### PR TITLE
bump: wl_output least supported version - 4

### PR DESCRIPTION
**Pull request by Shinyzenith**
This helps to populate name field when retrieving OutputInfo.

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -191,8 +191,8 @@ impl crate::environment::MultiGlobalHandler<WlOutput> for OutputHandler {
         version: u32,
         _: DispatchData,
     ) {
-        // We currently support wl_output up to version 3
-        let version = std::cmp::min(version, 3);
+        // We currently support wl_output up to version 4
+        let version = std::cmp::min(version, 4);
         let output = registry.bind::<WlOutput>(version, id);
         let has_xdg;
         if let Some(xdg) = self.xdg_listener.as_ref().and_then(rc::Weak::upgrade) {
```
Comment (vberger) [inline src/output.rs:198]: If version 4 is available, is it still necessary to bind the xdg_output global?

Comment (Shinyzenith) [reply src/output.rs:198]: Afaik smithay bound to xdg output just to get name and description events which are now fired by wloutput in v4. Should I unbind xdg output?

Comment (vberger) [reply src/output.rs:198]: In that case, maybe adding a check on that if so that `has_xdg` is set to `false` if `version >= 4` ?

Comment (Shinyzenith) [reply src/output.rs:198]: OK will get on it asap.

Comment (Shinyzenith) [reply src/output.rs:198]: @vberger https://github.com/Smithay/client-toolkit/pull/259/commits/51a7bf9306a83a548f4a610280b6cdb6c3246074

**Review by vberger — APPROVED** [review]

**Pull request merged**